packages/transport/src/modules/messages/MessageController.ts

```diff
@@ -507,10 +507,10 @@ export class MessageController extends TransportController {
                 );
             }
         } else {
-            relationship = await this.relationships.getActiveRelationshipToIdentity(envelope.createdBy);
+            relationship = await this.relationships.getOnceActiveRelationshipToIdentity(envelope.createdBy);
 
             if (!relationship) {
-                throw TransportCoreErrors.messages.missingOrInactiveRelationship(envelope.createdBy.toString());
+                throw TransportCoreErrors.general.recordNotFound("Once active Relationship to Identity", envelope.createdBy.toString());
```

Comment:
I really don't like the wording "once active Relationship"..

Reply:
this would be easier like this when you simplify the above
```suggestion
                throw TransportCoreErrors.general.recordNotFound(Relationship, envelope.createdBy.toString());
```